packages/backend/src/apps/lettersg/actions/create-letter/index.ts

```diff
@@ -40,6 +43,27 @@ const action: IRawAction = {
         ],
       },
     },
+    {
+      label: 'Generate PDF',
```

Comment:
Talked to stacey, we can leave this as a single step instead of having a separate "generate PDF" step. For now, better to prevent confusion during pipe setup.

Let's document this edge case instead - e.g. "PDF generation is complex and may fail. If you retry a "create letter" step that failed due to PDF errors, a duplicate letter may be issued. Please ignore the earlier issued letter in your dashboard if this happens"

Reply:
is this going to be on the guide instead?